### PR TITLE
Robot localization tuning

### PR DESCRIPTION
**Pull request by amiller27**
This is now pretty stable (enough that waypoints_test passes nicely).  There's still a little more noise than I'd like, but it's a good start.

### CODE REVIEW

File: launch/waypoints_test.launch [hunk 1]
```diff
@@ -8,7 +8,9 @@
     </include>
 
     <!-- This parameter must be true, overrides whatever's in morse.yaml -->
-    <param name="sim/ground_truth_localization" value="true" />
+    <!--<param name="sim/ground_truth_localization" value="true" />-->
```
Comment (aftersomemath) [inline launch/waypoints_test.launch]: Is this parameter needed anymore? Goes for the comment above it as well.

**Comment by aftersomemath** [comment]
I ended up implementing the change to where the cmd_vel was published because I wanted to test. Let me know if you have a problem with them.

File: launch/waypoints_test.launch (hunk 1 above)
Comment (amiller27) [reply launch/waypoints_test.launch]: I didn't mean to include this change, but I think it's best to just remove the parameter from this launch file and use robot_localization.

Comment (aftersomemath) [reply launch/waypoints_test.launch]: I agree

**Comment by amiller27** [comment]
Everything's now working pretty nicely with the ground truth camera localization and the simulated altimeter.  Should be ready to merge.

**Review by aftersomemath — APPROVED** [review]

**Pull request merged**